### ISSUE TITLE
DNS shouldn't be in apps management search result

### ISSUE THREAD

**Issue by aspasskaya** [AdguardTeam/AdguardForAndroid#2505]
!Also, "Android OS" cannot be found in the "Apps Management" section.

![image](https://user-images.githubusercontent.com/43952607/52062585-d1a87d80-2581-11e9-9bd0-99e8035f59c0.png)

![image](https://user-images.githubusercontent.com/43952607/52062717-1af8cd00-2582-11e9-98e1-1e4aa7d9d705.png)

![image](https://user-images.githubusercontent.com/43952607/52064264-58128e80-2585-11e9-902a-4b01960234bc.png)

**Comment by Nzzyme** [comment]
I searched with "android os" and see the correct app "Android OS" in the results.
Searched with "adguard" and see three results "DNS", "Android OS" and "AdGuard" (**which needs to be fixed**).
Searched with "dns" and see the correct entry "DNS" in the search results.

I think the title of this issue is slightly misleading. The entry "DNS" should be available in Apps Management but what I think the user @aspasskaya is referring to is that "DNS" should not be shown in the search results when we do a search with the keyword "adguard" (or in his case "android os").

Another thing what I noticed was that when I did a search with the keyword "app", I am seeing so many search results of apps that do not have the three characters "app" in their name. It is perhaps showing the apps whose package name has the keyword "app" (that is AG is not just searching the app's common name as seen below the icon but also searches for the package name). is this how it was designed to search?

Test version used for above: AG 3.0.229 (Nightly).
